drivers/serial/usart_mcux_lpc.c

```diff
@@ -290,17 +292,17 @@ static void usart_mcux_lpc_config_func_0(struct device *dev);
 #endif
 
 static const struct usart_mcux_lpc_config usart_mcux_lpc_0_config = {
-	.base = (USART_Type *)DT_USART_MCUX_LPC_0_BASE_ADDRESS,
-	.clock_source = 0,
-	.baud_rate = DT_USART_MCUX_LPC_0_BAUD_RATE,
+	.base = (USART_Type *)DT_NXP_LPC_USART_USART_0_BASE_ADDRESS,
+	.clock_source = kCLOCK_FlexComm0,
+	.baud_rate = DT_NXP_LPC_USART_USART_0_CURRENT_SPEED,
```

Comment:
Why are there USART refactoring patches in a GPIO PR?

Reply:
will drop commit and move to a separate pull

Reply:
Thanks